mdaviz/mda_file_table_model.py

```diff
@@ -374,6 +382,11 @@ def setFields(self, fields):
 
     # ------------ reporting
 
+    # TODO - later: we never call plotField with fieldname. Remove choices_pretty?
```

Comment:
You make a point to remove `choices_pretty` from the return.  The only occurrences of `plotFields` are:

```bash
$ git grep plotFields
mdaviz/mda_file.py:            new_selection_field = new_tab_tableview.tableView.model().plotFields()[0]
mdaviz/mda_file_table_model.py:        ~plotFields
mdaviz/mda_file_table_model.py:            self.checkboxStateChanged.emit(self.plotFields()[0], det_removed)
mdaviz/mda_file_table_model.py:    def plotFields(self):
```